src/box/sql/vdbe.c

```diff
@@ -2125,6 +2125,19 @@ case OP_Count: {         /* out2 */
 	break;
 }
 
+/**
+ * Opcode: DropConstraint P1 * * P4 *
+ * Synopsis: Drop constraint from box.space[P1]
+ *
+ * Drop constraint named P4.z from space P1.
+ */
+case OP_DropConstraint: {
+	assert(pOp->p1 >= 0 && pOp->p4.z != NULL);
+	if (sql_drop_core_constraint(pOp->p1, pOp->p4.z) != 0)
```

Comment:
Given that this function is used in the single place and it isn't really complicated nor big, I would suggest to inline it here. Up to you.

Reply:
I tried to inline this function, but it got pretty unattractive with all the `goto` I had to put in there. I think it's best to leave things as they are.